mlrun/serving/routers.py

```diff
@@ -700,26 +702,14 @@ def _parallel_run(self, event, mode: str = ParallelRunnerModes.thread):
             )
         return results
 
-    def validate(self, request, method):
-        """Validate the event body (after preprocessing)
-
-        Parameters
-        ----------
-        request : dict
-            Event body.
-        method  : str
-            Event method.
-        Returns
-        -------
-        dict
-            Event body after validation
-
-        Raises
-        ------
-        Exception
-            `inputs` key not found in `request`
-        Exception
-            `inputs` should be of type List
+    def validate(self, request: dict, method: str):
+        """
+        Validate the event body (after preprocessing)
+        :param request: Event body.
+        :param method: Event method.
+        :return: The given Event body (request).
+        :raise: Exception `inputs` key not found in `request`
+                Exception `inputs` should be of type List
         """
```

Comment:
In reStructuredText you need to add a newline (enter key) between paragraphs for `sphinx` to parse it later (also see my edits).

```suggestion
        """
        Validate the event body (after preprocessing)
        
        :param request: Event body.
        :param method: Event method.
        
        :return: The given Event body (request).
        
        :raise Exception: If validation failed.
        """
```